translate/storage/mozilla_lang.py

```diff
@@ -116,6 +130,13 @@ def parse(self, lines):
     def serialize(self, out):
         if self.is_active or self.mark_active:
             out.write(b"## active ##\n")
+        for header in self._headers:
```

Comment:
The file has a single header, not multiple headers. That single header can have multiple lines. So I suggest to alter the variables names to reflect that.